dart/pom.xml

```diff
@@ -35,6 +35,18 @@
       <artifactId>compile-testing</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.parceler</groupId>
+      <artifactId>parceler-api</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+
+    <dependency>
+        <groupId>com.squareup</groupId>
+        <artifactId>javapoet</artifactId>
```

Comment:
Can we agree on either first splitting the artifacts or accepting the PR
with JavaPoet as a dependency ?
@f2prateek ?

2015-05-16 12:18 GMT-07:00 Prateek Srivastava <notifications@github.com>:

> In dart/pom.xml
> <https://github.com/f2prateek/dart/pull/33#discussion_r30463728>:
>
> > @@ -35,6 +35,18 @@
> >        <artifactId>compile-testing</artifactId>
> >        <scope>test</scope>
> >      </dependency>
> > +    <dependency>
> > +      <groupId>org.parceler</groupId>
> > +      <artifactId>parceler-api</artifactId>
> > +      <scope>test</scope>
> > +    </dependency>
> > +
> > +
> > +    <dependency>
> > +        <groupId>com.squareup</groupId>
> > +        <artifactId>javapoet</artifactId>
>
> I don't use proguard, which is why the goal was to make the code
> generation part free of dependencies.
>
> I'm not entirely sure provided would work here, since the annotation
> processing happens when a user includes the dart jar in their classpath. By
> marking javapoet as provided, javapoet won't be available on the classpath
> when the user is compiling their app.
>
> I'm also working on splitting the artifacts for v2
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/f2prateek/dart/pull/33/files#r30463728>.
>